server/apiCalls/getRawAnalysisData.js

```diff
@@ -2,18 +2,36 @@
 const log = require('@kth/log')
 const api = require('../api')
 
+async function rawAnalysisDataFromCanvas(courseCode) {
+  try {
+    const { client, paths } = api.kursutvecklingApi
+    const kursutvApiResponse = await client.getAsync(
+      client.resolve(paths.getCanvasAnalysisListByCourseCode.uri, { courseCode }),
+      { useCache: true }
+    )
+    return kursutvApiResponse.body
+  } catch (error) {
+    const apiError = new Error('Kursutvecklingsinformation är inte tillgänlig för nu, försöker senare')
+    // apiError.status = 500
+    log.error('Error in getKursutvecklingApiInfo', { error })
+    throw apiError
+  }
+}
 
-async function rawAnalysisData (courseCode) {
+async function rawAnalysisDataFromKursinfoadmin(courseCode) {
   try {
     const { client, paths } = api.kursutvecklingApi
-    const kursutvApiResponse = await client.getAsync(client.resolve(paths.getAnalysisListByCourseCode.uri, { courseCode }), { useCache: true })
+    const kursutvApiResponse = await client.getAsync(
+      client.resolve(paths.getKursinfoadminAnalysisListByCourseCode.uri, { courseCode }),
+      { useCache: true }
+    )
     return kursutvApiResponse.body
```

Comment:
Nothing to do with this PR really as you just using the same approach as was here before, but one thing i noticed here is that the error handling isn't great. 

We are never checking status code of response. In my case I accidentally had set a incorrect api key and therefore got 401 here, but the body was returned anyway and the code failed later (in the sort function) because the body wasn't on expected format.

Reply:
Just to be clear, you don't have to fix this now! 

I'm just pointing it out to add to our collective knowledge that we have this behaviour and that we probably want to do something with at some point.

Reply:
This is true. We will have to add better error handling for this. I'll add an issue about it!

Reply:
Issue added here: https://kth-se.atlassian.net/browse/KUI-1605